server/webdriver/shared_tests/scatter_test.py

```diff
@@ -30,6 +29,10 @@
     'Count_Person_AsianAlone&svpy=0-14-1&svdy=Count_Person&svny=Asian_Alone&pcy=1'\
     '&epd=geoId/06&epn=California&ept=County'
 
+# Scatter plots can take extra long to load
+# This is a custom, longer timeout to use for charts we know are slow
+LONG_TIMEOUT = 120  # seconds
```

Comment:
let's move this to the shared library so all the timeout constants are together

Reply:
Good idea. Done!